ulas/chain.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "ULAS",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "ULAS",
+  "chain_id": "ulas",
+  "bech32_prefix": "ulas",
+  "daemon_name": "ulasd",
+  "node_home": "$HOME/.ulas",
+  "genesis": {
+    "genesis_url": "https://raw.githubusercontent.com/ulasnetwork/chain-config/main/ulas/genesis.json"
+  },
+  "codebase": {
+    "git_repo": "https://github.com/ulasnetwork/ulas",
+    "recommended_version": "v0.0.1",
+    "compatible_versions": ["v0.0.1"]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "http://65.49.204.199:26657",
+        "provider": "UlasNetwork"
+      }
+    ],
+    "rest": [
+      {
+        "address": "http://65.49.204.199:1317",
+        "provider": "UlasNetwork"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "http://65.49.204.199:9090",
+        "provider": "UlasNetwork"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "mintscan",
+      "url": "https://mintscan.io/ulasnetwork",
```

Comment:
This doesn't work on Mintscan. Is this supposed to be added soon? Or is there no explorer?

Reply:
updated to https://testnet-explorer.ulas.network